applications/user-generated-content.md

```diff
@@ -0,0 +1,25 @@
+---
+description: User-generated content in websites and social media
+---
+
+# User-Generated Content
+
+**User-generated content (USG)** is content created and shared through websites and social media. Machine translation is used in UGC mainly for gisting purposes.
```

Comment:
Machine translation for **user-generated content** (UGC) is used mainly for [gisting](/gisting).

Reply:
Let's not define what UGC is, that's out of scope.